function criaErro(): void {
    throw new Error('Erro criado');
}

criaErro();